### DISCUSSION TITLE
2 sd card rom combine

### DISCUSSION THREAD

**Discussion by spidicey** [rg35xx-cfw/Koriki#64] [Ideas]
can we combine roms from 2 sd card? i noticed that when there is the second sd card then the roms from the first one will not load

**Comment by acmeplus** [comment]
There's currently no such a functionality in the system. You could technically do it in koriki but you would have to 1) mount the first card roms partition into a given path, and 2) add the path to those roms into the .simplemenut/section_groups sections.
